grive/src/main.cc

```diff
@@ -196,14 +197,21 @@ int Main( int argc, char **argv )
 	Drive drive( &syncer, config.GetAll() ) ;
 	drive.DetectChanges() ;
 
-	if ( vm.count( "dry-run" ) == 0 )
+	if ( vm.count( "dry-run" ) == 0 && vm.count( "move" ) == 0 )
 	{
 		drive.Update() ;
 		drive.SaveState() ;
 	}
+  else if( vm.count( "move" ) > 0 ){
+    bool success = drive.Move();
+    if (success)
+      Log( "Move successful!", log::info );
+    else
+      Log( "Move failed.", log::error);
+  }
```

Comment:
Sorry, been trying out a new editor and I'm still figuring out how it handles indents. I'll fix this.